Makefile

```diff
@@ -37,10 +37,21 @@ source_path := ${build_dir}/SOURCES/${source_name}.tar.bz2
 all : prepare binary test rpm
 rpm: rpm_package_source rpm_build_source rpm_build
 
+python:
```

Comment:
Could be optimized.

Reply:
yah i wouldn't stress on any of the RPM or pyinstaller stuff.  It's going to be ripped out in full in CSM 1.3.